gateway/exporters/statsd/statsd.go

```diff
@@ -49,6 +49,7 @@ type StatsdExporter struct {
 	config  *configuration.GatewayConfig
 	connMgr *connections.ConnectionManager
 	client  statsd.Statter
+	logger  *fluent.Fluent
```

Comment:
let's rename this to fluentLogger, otherwise it can easily be confused with the zerolog one.